pulpcore/cli/ansible/repository.py

```diff
@@ -65,6 +64,17 @@
         "Remote used for synching in the form '[[<plugin>:]<resource_type>:]<name>' or by href."
     ),
 )
+repository_option = resource_option(
```

Comment:
I'm wondering if calling these `repository_lookup_option` would prevent confusion with the `repository_option` we use e.g. with distribution.

Reply:
I would leave it as it is. I consider it to be the same option like we use for distributions. Were you referring to something else?

```
pulp file repository create --name bbb
pulp file repository show --repository bbb
pulp file distribution create --name bbb --base-path bbb --repository bbb
```

Reply:
It should look the same on the outside, yes. But to the cli code inside it is something quite different. Maybe to make it more obvious, my suggestion is:
```suggestion
repository_lookup_option = resource_option(
```